GitHub-Test-DHprojects/ZackDGitHubTest4.md

```diff
@@ -0,0 +1,6 @@
+# GitHub Test 4
+
+The Blues Analysis Project investigated a collection of jazz music and sought out to answer multiple questions. Some of these questions were similarities and patterns between artists, connections and repeat lyrics that show up more than once across multiple songs and artists, and a look at the timelines of the artists, their songs, and their peers. Through their investigations they found multiple connections between data. They looked at noun usage across multiple artists and found many connections and patterns of frequency. 
+
+
+I really like the way in which they conducted the searches. It would be interesting to do different language / lyric comparisons between other artists or sources of media. Also the visual representation of the connected artists showed a really dynamic and comprehensible. With just a quick glance you can see centers of the culture and genre. In another project I feel like this could be a very valuable tool.
```

Comment:
Hello ztd5049,

I think this Markdown file is very well-formed, and I like how you are organizing your thoughts.

Harrison